vladbruma/equation-solver.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+class QuadraticSolver
+{
+
+    private $equation;
+    private $a = 0;
+    private $b = 0;
+    private $c = 0;
+    private $x1 = null;
+    private $x2 = null;
+
+    public function __construct($equation = null)
+    {
+        $this->equation = $equation;
+    }
+
+    public function filter()
+    {
+        $this->equation = strtolower(str_replace([' ', '*'], '', $this->equation));
```

Comment:
Why do you replace the * ?

Reply:
In case the multiply operation is defined by that symbol, like '4*x' instead of just '4x'